integration/tests/cook/test_multi_user.py

```diff
@@ -232,35 +232,50 @@ def test_preemption(self):
                 all_job_uuids.append(uuid_small)
                 instance = util.wait_for_running_instance(self.cook_url, uuid_small)
                 hostname = instance['hostname']
-                cpus = util.slave_cpus(self.mesos_url, hostname)
+                host_cpus = util.slave_cpus(self.mesos_url, hostname)
+                max_cpus = util.task_constraint_cpus(self.cook_url)
+                desired_cpus = host_cpus - small_cpus
+                if max_cpus >= desired_cpus:
+                    job_cpus = [desired_cpus]
+                else:
+                    num_max_cpu_jobs = int(desired_cpus / max_cpus)
+                    job_cpus = [max_cpus] * num_max_cpu_jobs
+                    job_cpus.append(desired_cpus - num_max_cpu_jobs * max_cpus)
 
                 # Submit a lower-priority job that fills the rest of the agent
```

Comment:
Can we fix the comment, since there can be multiple jobs?

Reply:
Done